packages/frontend/src/providers/TrackingProvider.tsx

```diff
@@ -277,8 +278,11 @@ export const TrackPage: FC<PageData> = ({ children, ...rest }) => {
     );
 };
 
-export const TrackSection: FC<SectionData> = ({ children, ...rest }) => (
-    <NestedTrackingProvider section={rest}>
-        {children || null}
-    </NestedTrackingProvider>
-);
+export const TrackSection: FC<SectionData> = memo(({ children, name }) => {
+    const section = useMemo(() => ({ name }), [name]);
+    return (
```

Comment:
if this fixes the issue, then great. Why do you think the `name` changes ? I  Would it be better to memo the whole `NestedTrackingProvider` component ?

Reply:
I think `...rest` has a different reference on each render, causing react to consider it a different value.